src/Model/Profile.php

```diff
@@ -785,6 +786,7 @@ public static function addVisitorCookieForHandle($handle)
 		$_SESSION['visitor_handle'] = $visitor['addr'];
 		$_SESSION['visitor_home'] = $visitor['url'];
 		$_SESSION['my_url'] = $visitor['url'];
+		$_SESSION['remote_follow'] = Probe::getRemoteFollowLink($visitor['url']);
```

Comment:
Shouldn't it be `remote_comment`?

Reply:
I had been undecided. Do you think this would be the better name?

Reply:
That's how you call it in the template.